ddnsd.go

```diff
@@ -94,7 +94,7 @@ func (s *dnsServer) dnsRemoveRequest(soa string, records []*pb.Record) (*dns.Msg
 		}
 		class, _ := dns.StringToClass["ANY"]
 		h := dns.RR_Header{Name: record.Domain, Rrtype: rrtype, Class: class, Ttl: 0}
-		rr, err := dns.NewRR(h.String())
+		rr, err := dns.NewRR(fmt.Sprintf("%s %s", h.String(), record.Data))
```

Comment:
I'm having trouble seeing how this is compliant with RFC2136.

The current code (before this PR) tries to do a "2.5.2 - Delete An RRset".

Now this is closest to "2.5.4 - Delete An RR From An RRset" but uses the wrong class.

I'm fine doing 2.5.4 but that requires changes to the Let's Encrypt integration we have.

Reply:
Changed to let client decide whether to do 2.5.2 or 2.5.4. Supplying record data will do 2.5.4